src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -324,6 +324,89 @@ void jit_exp_emitter::register_table_entries() {
     push_arg_entry_of("exponent_bias", 0x0000007f);
 }
 
+/// MOD ///
+jit_mod_emitter::jit_mod_emitter(ov::intel_cpu::riscv64::jit_generator* host, ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                 const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+}
+
+jit_mod_emitter::jit_mod_emitter(ov::intel_cpu::riscv64::jit_generator* host, ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                 const ov::element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+}
+
+size_t jit_mod_emitter::get_inputs_num() const {
+    return 2;
+}
+size_t jit_mod_emitter::aux_vecs_count() const {
+    if(exec_prc_ == ov::element::f32) {
+        return 1;
+    }
+    else if(exec_prc_ == ov::element::i32) {
+        return 0;
+    }
+    else {
+        OPENVINO_THROW("Unsupported precision");
+    }
+    
```

Comment:
Minor: fix code-style to align with other impls in the file (for now we have clang code style only on ARM64 and x64 platforms)
```suggestion
    if (exec_prc_ == ov::element::f32) {
        return 1;
    } else if (exec_prc_ == ov::element::i32) {
        return 0;
    } else {
        OPENVINO_THROW("Unsupported precision");
    }
```